src/app/kanban/page.tsx

```
@@ -109,9 +117,27 @@ function StandupPageContent() {
   const [loading, setLoading] = useState(true);
   const [refreshing, setRefreshing] = useState(false);
   const [error, setError] = useState<string | null>(null);
-  const [autoRefresh, setAutoRefresh] = useState(false);
-  const autoRefreshRef = useRef(autoRefresh);
-  autoRefreshRef.current = autoRefresh;
+  const [liveUpdate, setLiveUpdate] = useState(false);
+  const liveUpdateRef = useRef(liveUpdate);
+  liveUpdateRef.current = liveUpdate;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/app/kanban/page.tsx:122:3
  120 |   const [liveUpdate, setLiveUpdate] = useState(false);
  121 |   const liveUpdateRef = useRef(liveUpdate);
> 122 |   liveUpdateRef.current = liveUpdate;
      |   ^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  123 |   // Tracks whether we've finished reading the persisted value from
  124 |   // localStorage. The persist effect waits on this so its initial-mount run
  125 |   // (with the default `false`) can't overwrite a stored `'true'` before the

(react-hooks/refs)